### PR TITLE
Update all non ./mlflow docs

### PR DESCRIPTION
**Pull request by michael-berk**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/michael-berk/mlflow/pull/11110?quickstart=1)

#### Install mlflow from this PR

```
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/11110/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 11110
```

</p>
</details>

### What changes are proposed in this pull request?
* Convert to google docstring for all dirs in the root 
* Don't convert mleap - depricated formatter breaks docs build and formatting

https://github.com/mlflow/mlflow/pull/11078

### How is this PR tested?

- [x] Existing unit/integration tests
- [ ] New unit/integration tests
- [x] Manual tests

### Does this PR require documentation update?

- [x] No. You can skip the rest of this section.
- [ ] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [ ] Instructions

#### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/deployments`: MLflow Deployments client APIs, server, and third-party Deployments integrations
- [x] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: tests/utils/test_docstring_utils.py [hunk 1]
```diff
@@ -117,13 +117,11 @@ def test_param_docs_format_not_google():
     # fmt: off
     def f():
         """
-        asdf
-
-        :param p1: asdf
```
Comment (michael-berk) [inline tests/utils/test_docstring_utils.py:122]: Resolved

File: tests/tracking/fluent/test_fluent.py [hunk 2]
```diff
@@ -83,10 +83,12 @@ def create_run(
...
 def create_test_runs_and_expected_data(experiment_id=None):
-    """Create a pair of runs and a corresponding data to expect when runs are searched
-    for the same experiment
+    """
+    Create a pair of runs and a corresponding data to expect when runs are searched
+    for the same experiment.
 
-    :return: (list, dict)
+    Returns:
+        A tuple containing a list and a dictionary.
```
Comment (michael-berk) [inline tests/tracking/fluent/test_fluent.py:91]: Reasonable improvement

File: tests/sagemaker/mock/__init__.py [hunk 3]
```diff
@@ -316,7 +316,7 @@ def _url_module(self):
 
     def _get_base_arn(self, region_name):
         """
-        :return: A SageMaker ARN prefix that can be prepended to a resource name.
```
Comment (michael-berk) [inline tests/sagemaker/mock/__init__.py:319]: Resolved

File: tests/evaluate/test_validation.py [hunk 4]
```diff
@@ -226,13 +226,15 @@ def test_validation_faulty_validation_thresholds(
 def value_threshold_test_spec(request):
     """
     Test specification for value threshold tests:
-    :return: (
```
Comment (michael-berk) [inline tests/evaluate/test_validation.py:229]: Resolved

File: tests/evaluate/test_validation.py [hunk 5]
```diff
@@ -406,15 +408,15 @@ def test_validation_value_threshold_should_pass(
 def min_absolute_change_threshold_test_spec(request):
     """
     Test specification for min_absolute_change threshold tests:
-    :return: (
-                metrics: A dictionary mapping scalar metric names to scalar metric values,
```
Comment (michael-berk) [inline tests/evaluate/test_validation.py:410]: Resolved

File: tests/evaluate/test_validation.py [hunk 6]
```diff
@@ -616,15 +618,15 @@ def test_validation_model_comparison_absolute_threshold_should_pass(
 def min_relative_change_threshold_test_spec(request):
     """
     Test specification for min_relative_change threshold tests:
-    :return: (
```
Comment (michael-berk) [inline tests/evaluate/test_validation.py:619]: Resolved

File: mlflow/utils/docstring_utils.py [hunk 7]
```diff
@@ -134,21 +122,6 @@ def format_docstring(param_docs):
 
     Returns:
         A decorator to apply the formatting.
-
-    Examples:
```
Comment (michael-berk) [inline mlflow/utils/docstring_utils.py:138]: Resolved

File: mlflow/utils/docstring_utils.py [hunk 8]
```diff
@@ -97,21 +97,9 @@ def format_docstring(self, docstring: str) -> str:
         """
         Formats placeholders in `docstring`.
 
-        Examples
-        --------
-        >>> pd = ParamDocs(p1="doc1", p2="doc2
```
Comment (michael-berk) [inline mlflow/utils/docstring_utils.py:102]: Resolved

File: tests/tracking/fluent/test_fluent.py (hunk 2 above)
Comment (harupy) [reply tests/tracking/fluent/test_fluent.py:91]: Can we say `a tuple of a list and a dictionary`? `contain` sounds like it contains something else.

**Comment by harupy** [comment]
@michael-berk Thanks for the RP!

> Don't convert mleap - depricated formatter breaks docs build and formatting

Is this fixable or not?

**Review by harupy — APPROVED** [review]
LGTM!

**Pull request merged**